node_agent.spec

```diff
@@ -0,0 +1,67 @@
+%define pkg_name tendrl-node-agent
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
```

Comment:
This boilerplate duplication can be simplified:

```
Name: tendrl-node-agent
Version: 0.0.1
Release: 1%{?dist}
```

And then use `%{name}` and `%{version}` macros throughout the packaging, instead of the non-standard `%{pkg_name}` or `%{pkg_version}`